website/docs/API.md

```diff
@@ -791,9 +792,13 @@ function configure(options: Partial<Config>) {}
 
 Default timeout, in ms, for async helper functions (`waitFor`, `waitForElementToBeRemoved`) and `findBy*` queries. Defaults to 1000 ms.
 
+#### `defaultIncludeHiddenElements` option
+
+Default [includeHiddenElements](Queries.md#includehidden-option) query option for all queries. This default option will be overridden by the one you specify directly when using your query.
+
 #### `defaultHidden` option
```

Comment:
I would rather not give it a full header but rather put a mention in `defaultIncludeHiddenElements` entry that there is an alias for RTL compatibility.